Sources/MessagingPush/PushHandling/ManualPushHandling+UserNotifications.swift

```diff
@@ -59,11 +59,12 @@ extension MessagingPushImplementation {
         didReceive response: UNNotificationResponse,
         withCompletionHandler completionHandler: @escaping () -> Void
     ) -> Bool {
-        let push = UNNotificationWrapper(notification: response.notification)
-
-        guard push.isPushSentFromCio else {
-            // Exit early without calling completionHandler so that customer calls it instead.
-
+        // to keep this code DRY, forward the request to another function to perform all the logic:
+        guard let pushContent = userNotificationCenter(center, didReceive: response) else {
```

Comment:
had to think hard as to why might not it work, but i think its because of this, this was missed? 

```
if response.didClickOnPush {
            manualPushClickHandling(push: push)
        }
```